lizmap/modules/view/classes/lizmapMainViewItem.class.php

```diff
@@ -35,7 +35,7 @@ public function __construct($id, $title, $abstract = '', $keywordList = '', $pro
         $this->abstract = $abstract;
         $this->keywordList = $keywordList;
         $this->proj = $proj;
-        $this->bbox = $bbox;
+        $this->bbox = preg_replace('/([0-9]+\.[0-9]{2})[0-9]+/', '$1', $bbox);
```

Comment:
```suggestion
        $this->bbox = preg_replace('/([0-9]+\.[0-9]{6})[0-9]+/', '$1', $bbox);
```

Can you provide 6 decimals ?

Reply:
Not sure if I get the code at all... From a UX it would be nice to show 2 decimal places (rounded) for projected CRS, 6 I guess would be ok for geographical ones (degrees). Six decimals for CRSs in meters or feet do not give a good UX.